apps/frontend/src/pages/chat/libs/components/buttons-controller/libs/components/suggestions-creation-options/libs/enums/suggestions-creation-button-label.enum.ts

```diff
@@ -0,0 +1,6 @@
+const SuggestionsCreationButtonLabel = {
+	NO: "no, smth else",
+	YES: "yes, 3 lowest",
```

Comment:
```suggestion
	NO: "No, smth else",
	YES: "Yes, 3 lowest",
```

Reply:
But they are uppercased via css in the actual markdown